diff --git a/src/piper/__main__.py b/src/piper/__main__.py
--- a/src/piper/__main__.py
+++ b/src/piper/__main__.py
@@ -227,7 +227,7 @@
             with tempfile.NamedTemporaryFile("wb+", suffix=".wav") as temp_wav_file:
                 wav_file = wave.open(temp_wav_file.name, "wb")
                 lines_to_wav()
-
+                wav_file.close()
                 temp_wav_file.seek(0)
                 shutil.copyfileobj(temp_wav_file, sys.stdout.buffer)
         elif (not args.output_file) and AudioPlayer.is_available():
